packages/components/app/styles/components/stepper/indicator/step.scss

```diff
@@ -0,0 +1,184 @@
+//
+// STEPPER > INDICATOR > STEP
+//
+// properties within each class are sorted alphabetically
+//
+
+$hds-stepper-indicator-step-statuses: ( "incomplete", "progress", "processing", "complete" );
+$hds-stepper-indicator-step-size: 24px;
+
+// Base stepper indicator styling
+.hds-stepper-indicator-step {
+  width: $hds-stepper-indicator-step-size;
+  height: $hds-stepper-indicator-step-size;
+  position: relative;
+}
+
+.hds-stepper-indicator-step__svg-container {
+  filter: drop-shadow(0 1px 1px rgba(101, 106, 118, 0.05));
+  height: 100%;
+  width: 100%;
+  path {
+    fill: --status-fill-color;
+    stroke: --status-stroke-color;
+  }
+}
+
+.hds-stepper-indicator-step__status {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.hds-stepper-indicator-step__icon {
+  width: 12px;
+  height: 12px;
+  color: --status-text-color;
+}
+
+.hds-stepper-indicator-step__text {
+  font-family: var(--token-typography-font-stack-text);
+  font-weight: var(--token-typography-font-weight-medium);
+  font-size: 0.8125rem;
+  overflow: hidden;
+  text-align: center;
+  user-select: none;
+  white-space: nowrap;
+  width: 20px;
+  color: --status-text-color;
+}
+
+// STATUS
+
+// Non-interactive (default)
+
+// Non-interactive props that correspond with status
+$non-interactive-props: (
+  "incomplete": (
+    "text-color": var(--token-color-foreground-strong),
+    "fill-color": var(--token-color-surface-faint),
+    "stroke-color": var(--token-color-foreground-strong),
+  ),
+  "progress": (
+    "text-color": var(--token-color-foreground-high-contrast),
+    "fill-color": var(--token-color-palette-neutral-700),
+    "stroke-color": var(--token-color-palette-neutral-700),
+  ),
+  "processing": (
+    "text-color": var(--token-color-foreground-high-contrast),
+    "fill-color": var(--token-color-palette-neutral-700),
+    "stroke-color": var(--token-color-palette-neutral-700),
+  ),
+  "complete": (
+    "text-color": var(--token-color-foreground-high-contrast),
+    "fill-color": var(--token-color-palette-neutral-700),
+    "stroke-color": var(--token-color-palette-neutral-700),
+  )
+);
+
+@each $status in $hds-stepper-indicator-step-statuses {
+  // For each status of the non-interactive variant, set the text color, svg fill, and svg stroke colors based on $non-interactive-props
+  .hds-stepper-indicator-step--status-#{$status} {
+    .hds-stepper-indicator-step__text,
+    .hds-stepper-indicator-step__icon {
```

Comment:
[non-blocker/suggestion] I _think_ you can replace these two selectors with a single one: `hds-stepper-indicator-step__status` (also below in the file)

Reply:
Yep, worked fine!